module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,14 @@
+#include "sort.hpp"
+
+#include <algorithm>
+
+std::deque<std::string> lengthSort(std::forward_list<std::string> data) {
+    data.sort([](const std::string& lhs, const std::string& rhs) {
+        return (lhs.size() == rhs.size()) ? (lhs < rhs) : (lhs.size() < rhs.size());
```

Comment:
We did not allow to use ? : operator for safety critical software. It is not code review friendly.

Reply:
```suggestion
        return (lhs.size() == rhs.size()) ? (lhs < rhs)
                                          : (lhs.size() < rhs.size());
```

It is more friendly with this formatting